docs/rendering/modelextensions/facedata.md

```diff
@@ -0,0 +1,74 @@
+Face Data
+=========
+
+Additional data about an element's faces can be specified at either the element level or the face level in a vanilla "elements" model and in generated item models loaded through the `forge:item_layers` model loader. Faces which do not specify their own face data will fall back to the element's face data or a default if no face data is specified at the element level. All values of the face data are optional.
+
+In a vanilla "elements" model the additional data can be specified in the two ways shown in this example:
+```js
+{
+  "elements": [
+    {
+      "forge_data": {
+        "color": "0xFFFF0000",
+          "block_light": 15,
+          "sky_light": 15,
+          "ambient_occlusion": false
+      },
+      "faces": {
+        "north": {
+          "forge_data": {
+            "color": "0xFFFF0000",
+            "block_light": 15,
+            "sky_light": 15,
+            "ambient_occlusion": false
+          },
+          ...
+        },
+        ...
+      },
+      ...
```

Comment:
Use comments to explain things.

Also `// ...` to make valid JavaScript.

Reply:
I'm not sure what kind of comments to place within the example since the parameters are explained below